src/sessions/PageManager.ts

```diff
@@ -68,11 +68,18 @@ export class PageManager {
         this.page = undefined;
         this.resumed = false;
         this.recordInteraction = false;
-        this.virtualPageLoadTimer = new VirtualPageLoadTimer(
-            this,
-            config,
-            record
-        );
+        if (config.legacySPASupport) {
+            if (config.debug) {
+                InternalLogger.warn(
+                    'VirtualPageLoadTiming (deprecated) is enabled. Please use with caution after reviewing https://github.com/aws-observability/aws-rum-web/issues/723'
```

Comment:
comment: Should this message be a bit more explicit about the implication of it being enabled? 
For example, "Keeping this enabled will result in inaccurate page load timing with single page applications" 

I know you have linked the github issue, but perhaps something more explicit can make it clearer to the users of the implication of this and get them to disable it.